torchgeo/trainers/base.py

```diff
@@ -36,20 +36,20 @@ def __init__(self, ignore: Optional[Union[Sequence[str], str]] = None) -> None:
         """
         super().__init__()
         self.save_hyperparameters(ignore=ignore)
+        self.configure_models()
         self.configure_losses()
         self.configure_metrics()
-        self.configure_models()
+
+    @abstractmethod
+    def configure_models(self) -> None:
```

Comment:
Why is all this changed?

Reply:
See the PR description above

Reply:
Yes I read the PR description but that doesn't explain why you are moving the configure_models method around in the file. As far as I can tell that is unnecessary?

Reply:
Okay got it, seems fine!